database/buildSchema/29_review_response.sql

```diff
@@ -2,12 +2,16 @@
 
 CREATE TYPE public.review_response_decision as ENUM ('Approve', 'Decline');
 
+CREATE TYPE public.review_response_status as ENUM ('Draft', 'Submitted');
+
 CREATE TABLE public.review_response (
 	id serial primary key,
 	comment varchar,
-	decision public.review_response_decision,
+	review_response_decision public.review_response_decision,
 	review_question_assignment_id integer references public.review_question_assignment(id),
 	application_response_id integer references public.application_response(id),
+	review_response_link_id integer references public.review_response(id),
```

Comment:
I think it would just overcomplicate the schema, and won't be used. So I think is best to leave for when we actually implement this overrule of previous `review_decision`.